code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -190,6 +190,7 @@
 	new /obj/item/circuitboard/sleeper/hyper(src)
 	new /obj/item/oddity/chem_book(src)
 	new /obj/item/storage/hcases/med(src)
+	new /obj/item/tool/sword/saber/deconstuctive_rapier(src)
```

Comment:
```suggestion
	new /obj/item/tool/sword/saber/injection_rapier(src)
```

Under the impression CBO is suppose to get the medical one not the research one.